metrics/metrics.go

```diff
@@ -93,6 +94,11 @@ var (
 		TagKeys:     []tag.Key{KeyMessageType, KeyInstanceID},
 		Aggregation: defaultBytesDistribution,
 	}
+	TimeToFirstProviderView = &view.View{
+		Measure:     TimeToFirstProvider,
+		TagKeys:     []tag.Key{KeyMessageType, KeyPeerID, KeyInstanceID},
```

Comment:
`KeyMessageType` doesn't makes any sense in this particular case as it was meant to indicate which message type is being handled

Reply:
@lanzafame thanks